projects/plugins/jetpack/extensions/blocks/donations/edit.js

```diff
@@ -61,9 +60,8 @@ const Edit = props => {
 			return;
 		}
 
-		// Set fake products when plan should be upgraded or there is no connection to Stripe so users can still try the
-		// block in the editor.
-		if ( result.should_upgrade_to_access_memberships || result.connect_url ) {
+		// Set fake products when there is no connection to Stripe so users can still try the block in the editor.
+		if ( result.connect_url ) {
```

Comment:
@millerf  it's unclear if we want to make the Donation block free for all users as well

Reply:
Yes, all Earn products.